Sources/NIOCore/AsyncAwaitSupport.swift

```diff
@@ -240,8 +240,25 @@ extension ChannelPipeline {
     }
 }
 
+/// An error that is thrown when the number of bytes in an AsyncSequence exceeds the limit.
+///
+/// When collecting the bytes from an AsyncSequence, there is a limit up to where the content
+/// exceeds a certain threshold beyond which the content isn't matching the expected maximum size.
+/// that will be processed. This error is generally thrown when it is discovered that there are
+/// more bytes in a sequence than what was specified as the maximum. It could be that this upTo
+/// limit should be increased, or that the sequence has unexpected content in it.
+///
+/// - maxBytes: (optional) current limit on the maximum number of bytes in the sequence
 public struct NIOTooManyBytesError: Error, Hashable {
-    public init() {}
+    let maxBytes: Int?
+
+    public init() {
```

Comment:
We probably want to deprecate this constructor.

Reply:
It's deprecated now.